services/auth/repository/users.go

```diff
@@ -0,0 +1,41 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/Drozd0f/ttto-go/services/auth/schemes"
+	"github.com/Drozd0f/ttto-go/services/auth/db"
+	"github.com/lib/pq"
+)
+
+func (r *Repository) CreateUser(ctx context.Context, u *schemes.User) error {
+	err := r.q.CreateUser(ctx, u.UserToCreateUserParam())
+	if err != nil {
+		var pgError *pq.Error
+		if errors.As(err, &pgError) {
+			if pgError.Code == uniqueConstraintCode {
+				return fmt.Errorf("user exist: %w", ErrUniqueConstraint)
```

Comment:
I guess, here is no need for fmt.Errorf, add it for unknown error, and format it like
`fmt.Errorf("<who called> <what called>: %w", err)` for that case
`fmt.Errorf("queryes create user: %w", err)`